src/_type-scale.css

```diff
@@ -16,80 +16,68 @@
      7 = 7th step in size scale
 
    Media Query Extensions:
-     -ns = not-small
+     -s = small
      -m  = medium
      -l  = large
 */
 
-/*
- * For Hero/Marketing Titles
- *
- * These generally are too large for mobile
- * so be careful using them on smaller screens.
- * */
-
-.f-6,
-.f-headline {
-  font-size: 6rem;
-}
-.f-5,
-.f-subheadline {
-  font-size: 5rem;
-}
-
-
 /* Type Scale */
```

Comment:
Changing the type scale is the biggest breaking change in this PR, IMO. I hope this was well-considered. It makes sense though.